docs/yaml_config.rst

```diff
@@ -80,17 +90,18 @@ middlewares each exported topic needs to be forwarded to.
 Types definition
 ^^^^^^^^^^^^^^^^
 
-Some System Handles have the hability to give to soss the types definition
+Some System Handles have the ability to inform soss of the types definition
 (using `XTypes <https://github.com/eProsima/xtypes>`__) that they can use.
-`ros1` or `ros2` are examples of this.
-Nevertheless, there are cases where the System Handle is not able to get the type specification
+The System Handles of ROS1 and ROS2 are examples of this.
+Nevertheless, there are cases where the System Handle is not able to resolve the type specification
```

Comment:
```suggestion
Nevertheless, there are cases where the System Handle is not able to get the type specification
```

Reply:
get. from where?
I think I like resolve, but I don't really know.

Reply:
From itself. The types aren't resolved. Maybe `retrieve`?

Reply:
ok, retrieve.